lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -105,7 +110,8 @@ def build_container(machine_spec, machine_options)
         target_repository = 'chef'
```

Comment:
Yeah, I made a note on #27 - it could be out of scope for this PR on removing `docker_options[:from_image]`.  When we do it we will have to make sure we don't break anyone for backwards compatibility, otherwise we won't be able to change it until the next major version (1.0).